src/components/screens/wallet/delegateProfile/delegateVotesView.js

```diff
@@ -13,23 +13,25 @@ import styles from './delegateProfile.css';
 const DelegateVotesView = ({
   voters, t,
 }) => {
-  const [searchedAddress, setSearchedAddress] = useState('');
+  const [searchInput, setSearchInput] = useState('');
 
   const onInputChange = (e) => {
-    setSearchedAddress(e.target.value);
+    setSearchInput(e.target.value);
   };
 
   const handleLoadMore = () => {
     voters.loadData({ aggregate: true, offset: voters.meta.count + voters.meta.offset });
   };
 
-  const votersInfo = searchedAddress
-    ? voters.data.votes.filter(v => v.address === searchedAddress)
+  const votersInfo = searchInput
+    ? voters.data.votes.filter(
+      v => v.username.includes(searchInput) || v.address.includes(searchInput),
```

Comment:
if `username` or `address` is undefined then the code breaks just as it did when I checked it against testnet

![image](https://user-images.githubusercontent.com/32064769/152009147-50bca707-b65c-471d-8eb3-86758740c474.png)

```suggestion
      v => (v.username && v.username.includes(searchInput)) || (v.address && v.address.includes(searchInput)),
```